woocommerce/woocommerce-android
# Issue/5944 Preparation for Canada IPP tracking

Closes: #5944 


### Description

The PR contains small refactoring and adds a country code to every 

Sorry for the amount of changes, this is just class renaming

### Testing instructions

* Use debugger on `CardReaderTracker::track`
* Notice that country is added to the properties

### Images/gif

No UI changes
